Snakefile

```diff
@@ -0,0 +1,16 @@
+import os
+import glob
+
+# obtaining plate_ids
+sql_paths = glob.glob("./data/*.sqlite")
+PLATE_IDS = [Path(sql_file).stem for sql_file in sql_paths]
```

Comment:
You need to import `pathlib` here, right? Or is this something magic specific to the Snakefile?

Reply:
yes! completely forgot to remove the the `os` module and replace `pathlib` 

changes applied on commit: 21305ee51b64193213e31d112de1f6e5e17eff03